components/ModEditSubmissionForm.vue

```diff
@@ -692,61 +745,133 @@ const submissionHasUnsavedChanges = () => {
-                    website: submissionFormFields.website.value || '',
+                    googleMapsUrl: submissionFormFields.googlemapsURL.value,
+                    email: submissionFormFields.email.value,
+                    phone: submissionFormFields.phone.value,
+                    website: submissionFormFields.website.value,
                     address: {
-                        postalCode: submissionFormFields.postalCode.value || '',
-                        prefectureEn: submissionFormFields.prefectureEn.value || '',
-                        cityEn: submissionFormFields.cityEn.value || '',
-                        addressLine1En: submissionFormFields.addressLine1En.value || '',
-                        addressLine2En: submissionFormFields.addressLine2En.value || '',
-                        prefectureJa: submissionFormFields.prefectureJa.value || '',
-                        cityJa: submissionFormFields.cityJa.value || '',
-                        addressLine1Ja: submissionFormFields.addressLine1Ja.value || '',
-                        addressLine2Ja: submissionFormFields.addressLine2Ja.value || ''
+                        postalCode: submissionFormFields.postalCode.value,
+                        prefectureEn: submissionFormFields.prefectureEn.value,
+                        cityEn: submissionFormFields.cityEn.value,
+                        addressLine1En: submissionFormFields.addressLine1En.value,
+                        addressLine2En: submissionFormFields.addressLine2En.value,
+                        prefectureJa: submissionFormFields.prefectureJa.value,
+                        cityJa: submissionFormFields.cityJa.value,
+                        addressLine1Ja: submissionFormFields.addressLine1Ja.value,
+                        addressLine2Ja: submissionFormFields.addressLine2Ja.value
                     }
                 },
                 healthcareProfessionalIds: [],
                 mapLatitude: parseFloat(submissionFormFields.mapLatitude.value) || 0,
                 mapLongitude: parseFloat(submissionFormFields.mapLongitude.value) || 0
-            }
+            },
+            healthcareProfessionals: [
+                {
+                    acceptedInsurance: submissionFormFields.healthcareProfessionalAcceptedInsurances.value,
+                    degrees: submissionFormFields.healthcareProfessionalDegrees.value,
+                    specialties: submissionFormFields.healthcareProfessionalSpecialties.value,
+                    spokenLanguages: submissionFormFields.healthcareProfessionalLocales.value,
+                    names: submissionFormFields.healthCareProfessionalNameArray.value,
+                    facilityIds: []
+                }
+            ]
         }
     }
+    try {
+        const result = await moderationSubmissionStore.updateSubmission(submissionInputVariables)
+        const submissionResult = result as { updateSubmission: Submission }
```

Comment:
we should check for errors returned from the server

Reply:
is this not what the catch block is doing?

Reply:
no, that's a failure in the http request and shouldn't catch validation/business logic errors from the server. those are sent via the response object. 
Why? As a practice, you want to avoid exceptions as your error handling for planned errors (like bad user input, validation issues, etc) as they are slow and heavyweight, and should be reserved for errors we can't code against

Reply:
We talked about this